manifests/package/debian.pp

```diff
@@ -36,24 +37,27 @@
 
     case $package_source {
       'nginx', 'nginx-stable': {
+        $stable_repo_source = $repo_source ? { undef => "https://nginx.org/packages/${distro}", default => $repo_source }
```

Comment:
can you please use the multiline selector statement? Otherwise the lines get quite long

```suggestion
        $stable_repo_source = $repo_source ? {
          undef => "https://nginx.org/packages/${distro}",
          default => $repo_source
        }
```